build.sbt

```diff
@@ -20,9 +36,9 @@ val hmrcTestVersion            = "3.0.0"
 val scalatestVersion           = "3.0.0"
 val pegdownVersion             = "1.6.0"
 val jsoupVersion               = "1.10.2"
-val mockitoVersion             = "2.7.17"
+val mockitoVersion             = "2.13.0"
 val scalaMockVersion           = "3.5.0"
-val wiremockVersion            = "2.5.1"
+val wiremockVersion            = "2.6.0"
 
 val compile = Seq(
   ws,
```

Comment:
Can you add a return type to the `test(scope: String)` function to supress the warning.